src/background.js

```diff
@@ -1,3 +1,31 @@
+
+/*
+   Originally from Schalk Neethling at https://github.com/schalkneethling/dnt-helper
+ * Respects user choice & honor DoNotTrack
+ * Returns true or false based on whether doNotTack is enabled.
+ * @returns {boolean} true if enabled else false
+ */
+
+
+// browser.runtime.getManifest().version;
+
+function _dntEnabled(dnt, userAgent) {
+  'use strict';
+  let dntStatus = dnt || navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack;
+  const ua = userAgent || navigator.userAgent;
+  const fxMatch = ua.match(/Firefox\/(\d+)/);
+  var platform = ua.match(/Windows.+?(?=;)/g);
```

Comment:
we definitely don;t need to worry about platform compatibility, can you remove all the related code please?